docs/operations/hotplug_interfaces.md

```diff
@@ -0,0 +1,136 @@
+# Hotplug Network Interfaces
+KubeVirt now supports hotplugging network interfaces into a running Virtual
+Machine Instance (VMI). Hotplug is only supported for interfaces using the
+`virtio` model connected through
+[bridge binding](http://kubevirt.io/api-reference/main/definitions.html#_v1_interfacebridge).
+
+When a VM having hotplugged interfaces is rebooted, the hotplugged interfaces
+will be attached to the restarted VM.
+
```

Comment:
In a previous version, you wrote about the ability to persist the hotplug interface.
Please consider stating it here.